Client/Frontend/Browser/User Scripts/RandomConfiguration.swift

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import CryptoKit
+
+/// Class that aids creating the same random values for any given eTLD.
+///
+/// Constructing this class the same eTLD+1 will result in the same random values
+/// (provided the same `sessionKey` is used)
+/// - Note: Any string can actually be used and this class can be used for more general purpose applications
+class RandomConfiguration {
+    /// This is used to encode the domain key (i.e. the eTLD+1).
+    ///
+    /// For farbling, this key should be the same for the lifecycle of the app. Hence we use a static variable.
+    private static let sessionKey = SymmetricKey(size: .bits256)
```

Comment:
Is this okay? For example, if the user keeps the app open indefinitely (let's assume months), is it fine to keep using the same key?